project-resources/PROJECT_FUND.md

```diff
@@ -0,0 +1,68 @@
+# OpenJS Foundation Project-specific Initiative Funds
+
+The OpenJS Foundation (OpenJSF) provides a set of monetary funds for projects of the OpenJSF.
+These funds are designed to support collaboration and incentive project-specific initiatives that have high-impact within the individual project's scope.
+
+Alongside these lines, the fund exists to support the development of the project when the volunteer-based approach is not enough to support the community, e.g. infrastructure, tools, or security.
+
+## Archive of Past Project Funds
+
+The list below contains a historical archive and reference of past approved project fund requests.
+
+- [2023](../PROJECT_FUND/2023.md)
+
+## F.A.Q.
+
+### How might a project request funds?
+
+Funds should be requested on an ad-hoc basis through a PR requesting the funds. The requests should follow a specific template,
+but the description should contain relevant information regarding the project initiative,
+and any information that might be useful to highlight the need of funds and their benefits.
+
+The Pull Request template below should be completed with a Pull Request containing an addition to the `PROJECT_FUND/{CurrentYear}.md`
+spreadsheet from this repository, filling the required information.
+
+#### Fund Request PR Template
+
+```md
+### Fund Request for {Project Name}
+
+- **Purpose of the Request:**
+- **Intended usage of the resouces:**
+- **Success Criteria:**
+- **Total amount of funds requested:**
+
+## Pre-Request Evaluation
+
+#### Distribution of the Funds
+
+<!-- Explain here how these funds will be distributed/used -->
+
+#### Timeline of the Request
+
+<!-- Explain (possibly with a bullet-list) the timeline of the request, and how/when each part of the funds are being used -->
+
+## Post-request Mortem
+
+<!-- This should be written ad-hoc after the timeline was concluded -->
+
+#### How was the money spent
+
+<!-- How did the money get spent; did it follow the timeline? What diverged? -->
+
+---
+
+- [ ] **Was the success criteria met?**
+  - If not, please explain why and if possible any learnings.
```

Comment:
This post-mortem section probably doesn't belong in the template? As its own comment states, it should be ad-hoc, and written afterwards. If these sections are included in the PR's description, adding their contents later will not provide any notification to the PR's listeners.

It might be more appropriate to recommend or require adding a comment on the PR later with this information.